ui/app/AppLayouts/Chat/ChatLayout.qml

```diff
@@ -156,11 +234,11 @@ StackLayout {
             walletAssetsStore: root.walletAssetsStore
             currencyStore: root.currencyStore
             sendModalPopup: root.sendModalPopup
-            sectionItemModel: root.sectionItemModel
+            sectionItemModel: root.sectionItemWithMembers
             amIMember: sectionItem.amIMember
-            amISectionAdmin: root.sectionItemModel.memberRole === Constants.memberRole.owner ||
-                             root.sectionItemModel.memberRole === Constants.memberRole.admin ||
-                             root.sectionItemModel.memberRole === Constants.memberRole.tokenMaster
+            amISectionAdmin: root.sectionItemWithMembers.memberRole === Constants.memberRole.owner ||
```

Comment:
Here, you don't even need the member model(s), I think you can continue using the old `sectionItemModel`

Reply:
(and similar elsewhere in this file)